templates/rust/src/services/service.rs.twig

```diff
@@ -0,0 +1,47 @@
+use crate::client::{Client, ParamType};
+use std::collections::HashMap;
+use crate::services::{{spec.title | caseUcfirst}}Exception;
+
+#[derive(Clone)]
+pub struct {{ service.name | caseUcfirst }} {
+  client: Client
+}
+
+impl {{ service.name | caseUcfirst }} {  
+    pub fn new(client: &Client) -> Self {
+        Self {
+            client: client.clone()
+        }
+    }
+{% for method in service.methods %}
+
+{% if method.description %}
+{{ method.description|dartComment }}
+{% endif %}
+    pub fn {{ method.name | caseSnake }}(&self{% if method.parameters.all|length >= 1 %}, {% endif %}{% for parameter in method.parameters.all %}{{ parameter.name | caseSnake | escapeKeyword}}: {{ parameter.type | typeName }}{% if not loop.last %}, {% endif %}{% endfor %}) -> Result<reqwest::blocking::Response, {{spec.title | caseUcfirst}}Exception> {
+        let path = "{{ method.path|replace({'{': '', '}': ""}) }}"{% for parameter in method.parameters.path %}.replace("{{ parameter.name | caseSnake }}", &{{ parameter.name | caseSnake }}){% endfor %};
```

Comment:
```suggestion
        let path = "{{ method.path|replace({'{': '', '}': ""}) }}"{% for parameter in method.parameters.path %}.replace("{{ parameter.name | caseCamel }}", &{{ parameter.name | caseSnake }}){% endfor %};
```
Because the Swagger spec, path variables are in camel case